docs/content/0.1.0/docs/developers/releasing-new-version.md

```diff
@@ -0,0 +1,69 @@
+---
+weight: 100
+title: "Releasing New Version"
+---
+
+DataJunction project publishes all of its backend services and the Python client library to [PyPI](https://pypi.org/):
+
+- datajunction-clients/python
+- datajunction-query
+- datajunction-server
+- datajunction-reflection (NOT YET IMPLEMENTED)
```

Comment:
We actually do publish the reflection service here: https://pypi.org/project/datajunction-reflection/

Reply:
Oh, indeed.

Reply:
Fixed.